e2e/test/scenarios/permissions/admin-permissions.cy.spec.js

```diff
@@ -597,6 +621,30 @@ describe("scenarios > admin > permissions", { tags: "@OSS" }, () => {
           ["readonly", "Unrestricted", "Yes"],
         ]);
       });
+      it("should show a modal when a revision changes while an admin is editing", () => {
```

Comment:
I originally planned on having this test be navigating between DBs. That can still happen, but this demonstrates that we still check revisions when navigating to DB pages